src/chat.rs

```diff
@@ -362,6 +359,25 @@ impl ChatId {
             .unwrap_or_default() as usize
     }
 
+    pub(crate) fn get_param(self, context: &Context) -> Result<Params, Error> {
+        let res: Option<String> = context
+            .sql
+            .query_get_value_result::<_, _>("SELECT param FROM chats WHERE id=?", params![self])?;
+        Ok(res
+            .map(|s| s.parse().unwrap_or_default())
+            .unwrap_or_default())
```

Comment:
IIUC `s.parse` invokes the FromStr impl of Params, but I can't find a Default impl for Params so what does the default end up being?  I'm somewhat surprised we can use default here and don't know what it ends up being.